architecture/core_concepts/routes.adoc

```diff
@@ -163,11 +163,25 @@ plug-in and finally into an HAProxy configuration:
 .HAProxy Router Data Flow
 image::router_model.png[HAProxy Router Data Flow]
 
-*Sticky Sessions*
+[[routes-sticky-sessions]]
+=== Sticky Sessions
 
-Implementing sticky sessions is up to the underlying router configuration. The
+Sticky sessions attempts to ensure that user traffic is directed to the same pod, because a web application will perform better if session data, database data, and others, is cached. For example, for a cluster with five back-end pods, and two load-balanced routers, to increase performance, you can ensure that the same pod receives the web traffic from the same web browser regardless of the router that handles it.
+
+Placing returning traffic onto the same pod cannot be guaranteed. However, you can use HTTP headers to set a cookie determining the pod used in the last session. When the application user hits the router the browser re-sends the cookie and knows where to send the traffic.
+
+[NOTE]
+====
+Cookies cannot be set on pass-through routes, because the HTTP traffic cannot be seen. Instead, a number based on the source IP is set, which picks the back-end.
+
+If back-ends change, the traffic could head to the wrong server, making it less sticky, and if you are using a load-balancer (which hides the source IP) the same number is set for all connections and traffic is sent to the same pod. 
+====
+
+As a cluster administrator, you can turn off the stickiness for pass-through routes separately from other connections, or turn off stickiness entirely.
+
+The
 default HAProxy template implements sticky sessions using the `balance source`
```

Comment:
This is not true any more... we only use 'balance source' by default for pass-through routes.  Other route types default to 'balance leastconn' (unless multiple weighted services are in the route in which case we default to 'balance roundrobin' so the weights get used... but I'm not sure if that nuance is worth mentioning here).

BUT the first sentence 'The default HAProxy template implements sticky sessions using the `balance source` directive' is not really correct.

There are really two things going on:
1) Does this connection have a cookie... if so, read it and send it to the same pod it went to last time if still around
2) No cookie (because a new session, or stickiness disabled, or passthrough) pick a pod.  The defaults for that are as described above.  But they can be overriden by the route owner or the global default can be changed by the router admin

And that may need clarification here.